tests/queries/0_stateless/03262_parquet_s3_metadata_cache.sql

```diff
@@ -0,0 +1,28 @@
+-- Tags: no-parallel, no-fasttest
```

Comment:
can you also add a few tests for `parquet_metadata_cache_max_entries`?

Reply:
That would have to be an integration test, maybe with 10s or 100's of parquet files. I can add it in another PR.

Reply:
If local files also benefited from metadata cache, an integration test wouldn't be needed I suppose. But doesn't look like we want to do it

Reply:
For local Parquet files, OS file cache will be in effect.